kinto/core/utils.py

```diff
@@ -32,8 +32,18 @@
     memcache = None
 
 
-def json_serializer(v, **kw):
-    return json.dumps(v, escape_forward_slashes=False)
+class json:
+    def dumps(v, **kw):
+        return rapidjson.dumps(v, bytes_mode=rapidjson.BM_NONE)
```

Comment:
Dropping the `kw` will introduce breaking changes... See https://github.com/Kinto/kinto/blob/b9b4eb047ff0edb1d70a95a14c8afc9630c30d7d/kinto/plugins/quotas/utils.py#L6

Reply:
Perhaps I'm misunderstanding but the file you reference imports stdlib json so it shouldn't be affected. I will make a change however to respect passed in kw, if any; grepping the source I don't see any instances where this happens right now.

import json

def record_size(record):
    # We cannot use ultrajson here, since the `separator` option is not available.
    canonical_json = json.dumps(record, sort_keys=True, separators=(",", ":"))
    return len(canonical_json)